learn_error.tsv

```diff
@@ -0,0 +1,101 @@
+iter	SurvivalAft:dist=Extreme
```

Comment:
catboost/learn_error.tsv is definitely wrong file path

Reply:
fixed